bmc_dump_entry.hpp

```diff
@@ -61,6 +61,14 @@ class Entry :
           const std::filesystem::path& file,
           phosphor::dump::OperationStatus status, std::string originatorId,
           originatorTypes originatorType, phosphor::dump::Manager& parent) :
+        phosphor::dump::EntryIfaces(
+            bus, objPath.c_str(),
+            phosphor::dump::EntryIfaces::action::emit_no_signals),
+        phosphor::dump::Entry(bus, objPath.c_str(), dumpId, timeStamp, fileSize,
+                              status, originatorId, originatorType, parent),
+        phosphor::dump::bmc_stored::FileIfaces(
+            bus, objPath.c_str(),
+            phosphor::dump::bmc_stored::FileIfaces::action::defer_emit),
```

Comment:
why are these coming here?

Reply:
I ran the new clang-format 16 and this is how it is formatted. Only with this CI passes